apps/angular/projection/src/app/component/teacher-card/teacher-card.component.ts

```diff
@@ -1,35 +1,58 @@
 import { Component, OnInit } from '@angular/core';
-import { FakeHttpService } from '../../data-access/fake-http.service';
+import {
+  FakeHttpService,
+  randTeacher,
+} from '../../data-access/fake-http.service';
 import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
 import { Teacher } from '../../model/teacher.model';
-import { CardComponent } from '../../ui/card/card.component';
+import {
+  CardComponent,
+  CardListItemDirective,
+} from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-teacher-card',
-  template: `<app-card
-    [list]="teachers"
-    [type]="cardType"
-    customClass="bg-light-red"></app-card>`,
-  styles: [
-    `
-      ::ng-deep .bg-light-red {
-        background-color: rgba(250, 0, 0, 0.1);
-      }
-    `,
-  ],
+  template: `
+    <app-card [list]="teachers" customClass="bg-light-red">
+      <img src="assets/img/teacher.png" width="200px" image />
+
+      <ng-template cardListItem let-student>
+        <app-list-item
+          [name]="student.firstName"
+          [id]="student.id"
+          (deleteItem)="onDeleteItem($event)"></app-list-item>
+      </ng-template>
+
+      <button
+        class="rounded-sm border border-blue-500 bg-blue-300 p-2"
+        (click)="addNewItem()">
+        Add
+      </button>
```

Comment:
You can leave this one inside the card component and use a output to trigger to add event. But that's up to you. Both implementations work